hw/drivers/nrf52_bluetooth/nrf52_bluetooth_internal.h

```diff
@@ -0,0 +1,16 @@
+/* nrf52_bluetooth_internal.h
+ * Definitions for internal components of nRF52 Bluetooth driver, including PPoGATT.  Not for external use.
+ * RebbleOS
+ *
+ * Author: Joshua Wise <joshua@joshuawise.com>
+ */
+#pragma once
+
+#include "ble_db_discovery.h"
+
+#define PPOGATT_MTU 256
```

Comment:
we should have a better mechanism for this. per platform MTU config

Reply:
Yeah, I agree.  PPOGATT MTU is still busted in a lot of ways; MTU negotiation doesn't take place properly, for instance.  I don't really understand how it's supposed to work, and it's going to take some time before I do, I think.  In the mean time, this is a ... maximum ... MTU.